src/storage/index/IndexPolicyMaker.cpp

```diff
@@ -41,41 +41,54 @@ cpp2::ErrorCode IndexPolicyMaker::decodeExpression(const std::string &filter) {
     return code;
 }
 
-void IndexPolicyMaker::buildPolicy() {
-    prefix_.reserve(256);
-    decltype(operatorList_.size()) hintNum = 0;
-    bool hasStr = false;
+bool IndexPolicyMaker::buildPolicy() {
```

Comment:
What's the difference with the original logic?
Add comments about this method.

Reply:
> What's the difference with the original logic?
> Add comments about this method.

original logic : if has string type, need to do expression filter.
current logic : don't need to  expression filter when has string type, because the query contains all column of index, and the scan string appended the string length.